pkgs/by-name/op/ophcrack-cli/package.nix

```diff
@@ -0,0 +1,78 @@
+{ lib
+, stdenvNoCC
+, libsForQt5
+, fetchFromGitLab
+, autoreconfHook
+, libtool
+, pkg-config
+, zlib
+, openssl
+, freetype
+, fontconfig
+, gcc
+, expat
+, cmake
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "ophcrack";
+  version = "3.8.0-3";
+
+  src = fetchFromGitLab {
+    domain = "salsa.debian.org";
+    owner = "pkg-security-team";
+    repo = "ophcrack";
+    rev = "debian/${version}";
+    hash = "sha256-94zEr7aBeqMjy5Ma50W3qv1S5yx090bYuTieoZaXFcc=";
+  };
+
+  nativeBuildInputs = [
+    autoreconfHook
+    libtool
+    libsForQt5.wrapQtAppsHook
+  ];
+
+  buildInputs = [
+    pkg-config
+    zlib
+    openssl
+    freetype
+    fontconfig
+    libsForQt5.qtcharts
+    gcc
+    expat
+  ];
+
+  dontWrapQtApps = true;
+
+  configureFlags = [
+    "--with-libssl=yes --disable-gui"
+  ];
+
+  env.CFLAGS = toString [
+    "-I/usr/include/libxml2/libxml"
```

Comment:
It looks wrong. Why are you doing this?

Reply:
You are correct, i was running the package build in different linux distro which required i pointed to libxml but it works perfectly without pointing to libxml when i packaged in Ubuntu without it. Thanks!